### DISCUSSION TITLE
How is the temperature calculated?

### DISCUSSION THREAD

**Discussion by TornaxO7** [ClementTsang/bottom#1305] [Q&A]
Hi! I'm wondering how the CPU temperature is calculated because I'm getting the following output:
![image](https://github.com/ClementTsang/bottom/assets/50843046/94c11359-044a-4ce1-a886-55a44740cac3)
I can't really believe that one core has only 15 degrees celcius.

**Comment by ClementTsang** [comment]
It's pulled straight from whatever your OS reports... unfortunately what the OS reports isn't always correct.

**Reply by TornaxO7** [reply]
Aw man... so you think that those temperatures are wrong as well?
